packages/playground/src/dashboard/components/public_ips_table.vue

```diff
@@ -119,21 +116,28 @@ export default {
     const isRemoving = ref(false);
     const selectedItems = ref<any[]>([]);
     const items = ref<RemoveFarmIPModel[]>([]);
+    const page = ref<number>(1);
+    const pageSize = ref(10);
 
-    onMounted(async () => {
-      await getFarmByID(props.farmId);
-    });
-
-    async function getFarmByID(id: number) {
+    async function getFarmPublicIp() {
       loadingIps.value = true;
       try {
-        const farm = await gridStore.grid.farms.getFarmByID({ id });
-        publicIps.value = farm.publicIps as unknown as PublicIp[];
+        const { data, count } = await gridProxyClient.publicIps.list({
+          retCount: true,
```

Comment:
Please make sure not to return count on each request. If I click on the next page, there is no need to return the count, this flag is very heavy

Reply:
please check this issue, https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1691; we should have the ret count only on the first request or whenever we need to get the count only